src/main/java/teammates/ui/pagedata/PageData.java

```diff
@@ -990,7 +990,7 @@ protected boolean isResponseVisibleTo(FeedbackParticipantType participantType, F
      * @param commentsForResponses map where key is response id and value is list of comments on that response
      * @param responseId response id of response associated with comment
      * @param isEditDeleteEnabled true if comment can be edited or deleted
-     * @return
+     * @return feedback response comment with associated parameters
```

Comment:
It is not `feedback response comment with associated parameters`. The `FeedbackResponseCommentRow` is a helper class for UI display. You can just remove the `@return`.